### PR TITLE
Fixes for TravisCI

### PR DESCRIPTION
**Pull request by jogibear9988**
[Travis CI builds](https://travis-ci.org/jogibear9988/linq2db/) 

badge

Issue #504

### CODE REVIEW

File: Source/global.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "sdk": { "version": "1.0.0-preview2-1-003177" }
```
Comment (ili) [inline Source/global.json]: @jogibear9988 are you sure this is needed? by default should be used latest installed also configured `dotnet: 1.0.0-preview2-003156`

Comment (jogibear9988) [inline Source/global.json]: i don't need this on travis, but on windows! without it uses the newer(i have vs2017 installed), and i could not compile

**Comment by ili** [comment]
About MySql, try to use MySql57 instead of MySql in *DataProviders.txt (it should solve problems with year).

File: Tests/Linq/TravisDataProviders.Osx.txt [hunk 2]
```diff
@@ -1,8 +1,3 @@
-﻿--Access
---SqlCe
---SQLite
---NorthwindSqlite
-
-!MySql           * Server=localhost;Database=TestData;Uid=root;Pwd=;charset=utf8;                                            * MySql.Data.MySqlClient
-!PostgreSQL     * Server=localhost;Database=testdata;User Id=postgres;Password=;Pooling=true;MinPoolSize=10;MaxPoolSize=100 * Npgsql
+﻿
+PostgreSQL     * Server=localhost;Database=testdata;User Id=postgres;Password=;Pooling=true;MinPoolSize=10;MaxPoolSize=100 * Npgsql
```
Comment (ili) [inline Tests/Linq/TravisDataProviders.Osx.txt]: `!` symbol means that it is default configuration if it is not set, then would be used `SQLite` or `SQLiteMs` Default connection is widely used in tests, so if we do not run `SQLite` we need to redefine one. So you should use !PostgreSQL

**Comment by ili** [comment]
Is it possible to make more complex travis matrix:

- linux
  - mono
  - core
- osx
  - mono
  - core

Now all tests are done in one buildlog, and if i'm not mistaken build fails after failing Core tests, it would be great if we can also see mono results...

**Comment by jogibear9988** [comment]
don't know... but i think, at first we should check that it works... i think if we splitt to 4 tasks, the build takes much longer, because they are not run in parallel

**Comment by ili** [comment]
OK, it looks like we do have `-1.79E+308d` problem... 
lets do one thing - 
* lets extract double tests with 1.7900000000000008E+308d in separate method and mark them as `WindowsOnly`
* in numeric tests lets try to make some cheat like `1.5E+308d` may be it would help...

**Comment by ili** [comment]
@jogibear9988 what problems we do have now with travis build?

**Comment by jogibear9988** [comment]
i'll look after the orbattle pull req works

**Comment by ili** [comment]
F# tests are failing on appveyor. Need to update nunit package reference in project.

**Pull request merged**